joomla/joomla-cms
# [4.0] Feature: Article > Save To Menu

Pull Request for Issue from facebook
https://www.facebook.com/groups/joomla.deutschland/permalink/1346440042196465/

### Summary of Changes
Save a content directy to a new menu item.

PR requires https://github.com/joomla/joomla-cms/pull/26675

Update: Removes also the old useless "add menu item" button.

### Testing Instructions
1. Apply Patch https://github.com/joomla/joomla-cms/pull/26675 (required)
2. Apply these Partch
3. Create a new article
4. Click on "Save to menu"
![savetomenu](https://user-images.githubusercontent.com/12803406/67144136-d3cb7d00-f273-11e9-9668-8f4946681437.jpg)
5. Article should been saved and redirects you to com_menu
6. Menu SingleArticle should been preselected with the article you've created and the title from the article


### Expected result
![MenuNew](https://user-images.githubusercontent.com/12803406/67144164-1bea9f80-f274-11e9-950d-1c14f3bbcdc5.jpg)

### Actual result
New Feature


### Documentation Changes Required
Yes, maybe

PR for the user survey from facebook + 50 users voted
![SaveContent2Menu](https://user-images.githubusercontent.com/12803406/67144181-42a8d600-f274-11e9-9e05-511d204af700.jpg)

### Additional Infos
**It's a global Save2Menu Function, could be finaly implement in any component**
